_data/authors.yml

```diff
@@ -1,41 +0,0 @@
-# Authors
```

Comment:
Those seem to be used here: `git grep 'author: ' ./_posts/`

Reply:
Oops, fixed. Thanks